README.markdown

```diff
@@ -6164,6 +6176,102 @@ See also [ngx.shared.DICT](#ngxshareddict).
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.shared.DICT.lpush
+---------------------
+**syntax:** *length, err = ngx.shared.DICT:lpush(key, value)*
+
+**context:** *init_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.**
+
+Inserts the specified (numerical or string) `value` at the head of the list named `key` in the shm-based dictionary [ngx.shared.DICT](#ngxshareddict). Returns the length of the list after the push operations.
```

Comment:
The "length of the list" is ambiguous. Is it a length in bytes or the number of elements? We should be clearer here. Better be "Returns the number of elements in the list after the push operation". Oh yeah, "operations" should be "operation" since it is a single operation, not multiple.